### PR TITLE
Changing (mostly) case study links from what-we-deliver to our-work

### PR DESCRIPTION
**Pull request by michelle-rago**
Updated case study links from what-we-deliver to our-work

[Review these changes](https://federalist-c58f58dc-a215-4616-a54c-12b5eb011096.sites.pages.cloud.gov/preview/18f/18f.gsa.gov/case-study-redirects/)

### CODE REVIEW

File: _posts/2017-06-01-federalist-is-out-of-beta-and-open-for-business.md
Comment (ameliaswong) [inline _posts/2017-06-01-federalist-is-out-of-beta-and-open-for-business.md:1]: @michelle-rago /our-work/federalist/ renders a page with the title "Federalist is now cloud.gov Pages" and copy noting this change—not the original case study. That's as expected, correct?

File: _posts/2020-04-09-18f-help-during-critical-times.md
Comment (ameliaswong) [inline _posts/2020-04-09-18f-help-during-critical-times.md:1]: @michelle-rago /our-work/#some-agencies-weve-worked-with/ loads the /our-work/ page, not the page at the anchor. As expected?

File: _posts/2017-06-01-federalist-is-out-of-beta-and-open-for-business.md
Comment (michelle-rago) [reply _posts/2017-06-01-federalist-is-out-of-beta-and-open-for-business.md:1]: Yes, I believe so. Thanks for checking!

File: _posts/2017-06-22-why-discovery-research-matters-for-modular-procurement.md
Comment (ameliaswong) [inline _posts/2017-06-22-why-discovery-research-matters-for-modular-procurement.md:1]: @michelle-rago /our-work/agile-bpa/ 404s for me. Is that something we can fix?

File: _products_projects/micro-purchase-marketplace.md
Comment (ameliaswong) [inline _products_projects/micro-purchase-marketplace.md:1]: @michelle-rago Also getting a 404 for /our-work/micro-purchase-marketplace/

File: _services_projects/dod-military-onesource.md
Comment (ameliaswong) [inline _services_projects/dod-military-onesource.md:1]: @michelle-rago Another 404 for /our-work/military-onesource/

File: _services_projects/doi-every-kid-in-a-park.md
Comment (ameliaswong) [inline _services_projects/doi-every-kid-in-a-park.md:1]: @michelle-rago Another 404: /our-work/every-kid-in-a-park/

File: _posts/2020-04-09-18f-help-during-critical-times.md
Comment (michelle-rago) [reply _posts/2020-04-09-18f-help-during-critical-times.md:1]: I guess the link in that post would have to be changed to https://18f.gsa.gov/our-work/#some-agencies-weve-worked-with, but I think it's okay that it goes to the top of /our-work/ or I have to do something more clever with the redirect. I think it's okay as is though. Let me know if you don't. 😄

File: _services_projects/navy-reserve.md
Comment (ameliaswong) [inline _services_projects/navy-reserve.md:1]: @michelle-rago One more 404: /our-work/ready-2-serve/

File: _services_projects/treasury-new-ten.md
Comment (ameliaswong) [inline _services_projects/treasury-new-ten.md:1]: @michelle-rago Another 404 for the new permalink.

**Comment by ameliaswong** [comment]
@michelle-rago Most of the revised links render, but a handful 404ed. I commented on each within the .md file to you—hope that's okay, but also apologies for the barrage of email that choice probably generated.

**Comment by michelle-rago** [comment]
Thanks @ameliaswong! I'm noticing that the filenames for these case studies are different from the named links in the .md files. I'm poking around to see what's up.

File: _products_projects/micro-purchase-marketplace.md
Comment (michelle-rago) [reply _products_projects/micro-purchase-marketplace.md:1]: I'm guessing this one isn't published. The front matter includes:

```
published: false
listed: false
```

File: _services_projects/dod-military-onesource.md
Comment (michelle-rago) [reply _services_projects/dod-military-onesource.md:1]: Same with this one:

```
published: false
listed: false
```

File: _services_projects/doi-every-kid-in-a-park.md
Comment (michelle-rago) [reply _services_projects/doi-every-kid-in-a-park.md:1]: And this one:

```
published: false
listed: false
```

File: _services_projects/navy-reserve.md
Comment (michelle-rago) [reply _services_projects/navy-reserve.md:1]: Here too:

```
published: false
listed: false
```

Comment (michelle-rago) [reply _services_projects/navy-reserve.md:1]: You guessed it!

```
published: false
listed: false
```

File: _services_projects/treasury-new-ten.md
Comment (michelle-rago) [reply _services_projects/treasury-new-ten.md:1]: published: false
listed: false

File: _posts/2017-06-22-why-discovery-research-matters-for-modular-procurement.md
Comment (michelle-rago) [reply _posts/2017-06-22-why-discovery-research-matters-for-modular-procurement.md:1]: I changed it to https://github.com/18f/ads-bpa

**Pull request merged**